[BUG] alluxio UT failed to build in Databricks runtime

**Describe the bug**
introduced in https://github.com/NVIDIA/spark-rapids/pull/7155


```
[2022-12-06T09:32:33.404Z] [INFO] --- scala-maven-plugin:3.4.4:testCompile (scala-test-compile-first) @ rapids-4-spark-sql_2.12 ---

[2022-12-06T09:32:33.404Z] [INFO] Using incremental compilation

[2022-12-06T09:32:33.404Z] [WARNING] Pruning sources from previous analysis, due to incompatible CompileSetup.

[2022-12-06T09:32:33.404Z] [INFO] Compiling 4 Scala sources to /home/ubuntu/spark-rapids/sql-plugin/target/spark312db/test-classes...

[2022-12-06T09:32:33.404Z] [ERROR] /home/ubuntu/spark-rapids/sql-plugin/src/test/scala/com/nvidia/spark/rapids/AlluxioUtilsSuite.scala:118: overloaded method value apply with alternatives:

[2022-12-06T09:32:33.404Z]   (values: org.apache.spark.sql.catalyst.InternalRow,files: Seq[org.apache.spark.sql.execution.datasources.SerializableFileStatus])org.apache.spark.sql.execution.datasources.PartitionDirectory <and>

[2022-12-06T09:32:33.404Z]   (values: org.apache.spark.sql.catalyst.InternalRow,files: Array[org.apache.hadoop.fs.FileStatus])org.apache.spark.sql.execution.datasources.PartitionDirectory

[2022-12-06T09:32:33.404Z]  cannot be applied to (Null, Seq[org.apache.hadoop.fs.FileStatus])

[2022-12-06T09:32:33.404Z] [ERROR]     val pds = Seq(PartitionDirectory(null, Seq(fs)))

[2022-12-06T09:32:33.404Z] [ERROR]                   ^

[2022-12-06T09:32:33.404Z] [ERROR] /home/ubuntu/spark-rapids/sql-plugin/src/test/scala/com/nvidia/spark/rapids/AlluxioUtilsSuite.scala:137: overloaded method value apply with alternatives:

[2022-12-06T09:32:33.404Z]   (values: org.apache.spark.sql.catalyst.InternalRow,files: Seq[org.apache.spark.sql.execution.datasources.SerializableFileStatus])org.apache.spark.sql.execution.datasources.PartitionDirectory <and>

[2022-12-06T09:32:33.404Z]   (values: org.apache.spark.sql.catalyst.InternalRow,files: Array[org.apache.hadoop.fs.FileStatus])org.apache.spark.sql.execution.datasources.PartitionDirectory

[2022-12-06T09:32:33.405Z]  cannot be applied to (Null, Seq[org.apache.hadoop.fs.FileStatus])

[2022-12-06T09:32:33.405Z] [ERROR]     val pds = Seq(PartitionDirectory(null, Seq(fs)))

[2022-12-06T09:32:33.405Z] [ERROR]                   ^

[2022-12-06T09:32:33.405Z] [ERROR] /home/ubuntu/spark-rapids/sql-plugin/src/test/scala/com/nvidia/spark/rapids/AlluxioUtilsSuite.scala:157: overloaded method value apply with alternatives:

[2022-12-06T09:32:33.405Z]   (values: org.apache.spark.sql.catalyst.InternalRow,files: Seq[org.apache.spark.sql.execution.datasources.SerializableFileStatus])org.apache.spark.sql.execution.datasources.PartitionDirectory <and>

[2022-12-06T09:32:33.405Z]   (values: org.apache.spark.sql.catalyst.InternalRow,files: Array[org.apache.hadoop.fs.FileStatus])org.apache.spark.sql.execution.datasources.PartitionDirectory

[2022-12-06T09:32:33.405Z]  cannot be applied to (Null, Seq[org.apache.hadoop.fs.FileStatus])

[2022-12-06T09:32:33.405Z] [ERROR]     val pds = Seq(PartitionDirectory(null, Seq(fs)))

[2022-12-06T09:32:33.405Z] [ERROR]                   ^

[2022-12-06T09:32:33.405Z] [ERROR] /home/ubuntu/spark-rapids/sql-plugin/src/test/scala/com/nvidia/spark/rapids/AlluxioUtilsSuite.scala:182: overloaded method value apply with alternatives:

[2022-12-06T09:32:33.405Z]   (values: org.apache.spark.sql.catalyst.InternalRow,files: Seq[org.apache.spark.sql.execution.datasources.SerializableFileStatus])org.apache.spark.sql.execution.datasources.PartitionDirectory <and>

[2022-12-06T09:32:33.405Z]   (values: org.apache.spark.sql.catalyst.InternalRow,files: Array[org.apache.hadoop.fs.FileStatus])org.apache.spark.sql.execution.datasources.PartitionDirectory

[2022-12-06T09:32:33.405Z]  cannot be applied to (Null, Seq[org.apache.hadoop.fs.FileStatus])

[2022-12-06T09:32:33.405Z] [ERROR]     val pds = Seq(PartitionDirectory(null, Seq(fs)))

[2022-12-06T09:32:33.405Z] [ERROR]                   ^

[2022-12-06T09:32:33.405Z] [ERROR] /home/ubuntu/spark-rapids/sql-plugin/src/test/scala/com/nvidia/spark/rapids/AlluxioUtilsSuite.scala:205: overloaded method value apply with alternatives:

[2022-12-06T09:32:33.405Z]   (values: org.apache.spark.sql.catalyst.InternalRow,files: Seq[org.apache.spark.sql.execution.datasources.SerializableFileStatus])org.apache.spark.sql.execution.datasources.PartitionDirectory <and>

[2022-12-06T09:32:33.405Z]   (values: org.apache.spark.sql.catalyst.InternalRow,files: Array[org.apache.hadoop.fs.FileStatus])org.apache.spark.sql.execution.datasources.PartitionDirectory

[2022-12-06T09:32:33.405Z]  cannot be applied to (Null, Seq[org.apache.hadoop.fs.FileStatus])

[2022-12-06T09:32:33.405Z] [ERROR]     val pds = Seq(PartitionDirectory(null, Seq(fs)))

[2022-12-06T09:32:33.405Z] [ERROR]                   ^

[2022-12-06T09:32:33.405Z] [ERROR] /home/ubuntu/spark-rapids/sql-plugin/src/test/scala/com/nvidia/spark/rapids/AlluxioUtilsSuite.scala:231: overloaded method value apply with alternatives:

[2022-12-06T09:32:33.405Z]   (values: org.apache.spark.sql.catalyst.InternalRow,files: Seq[org.apache.spark.sql.execution.datasources.SerializableFileStatus])org.apache.spark.sql.execution.datasources.PartitionDirectory <and>

[2022-12-06T09:32:33.405Z]   (values: org.apache.spark.sql.catalyst.InternalRow,files: Array[org.apache.hadoop.fs.FileStatus])org.apache.spark.sql.execution.datasources.PartitionDirectory

[2022-12-06T09:32:33.405Z]  cannot be applied to (Null, Seq[org.apache.hadoop.fs.FileStatus])

[2022-12-06T09:32:33.405Z] [ERROR]     val pds = Seq(PartitionDirectory(null, Seq(fs)))

[2022-12-06T09:32:33.405Z] [ERROR]                   ^

[2022-12-06T09:32:33.405Z] [ERROR] /home/ubuntu/spark-rapids/sql-plugin/src/test/scala/com/nvidia/spark/rapids/AlluxioUtilsSuite.scala:237: overloaded method value apply with alternatives:

[2022-12-06T09:32:33.405Z]   (values: org.apache.spark.sql.catalyst.InternalRow,files: Seq[org.apache.spark.sql.execution.datasources.SerializableFileStatus])org.apache.spark.sql.execution.datasources.PartitionDirectory <and>

[2022-12-06T09:32:33.405Z]   (values: org.apache.spark.sql.catalyst.InternalRow,files: Array[org.apache.hadoop.fs.FileStatus])org.apache.spark.sql.execution.datasources.PartitionDirectory

[2022-12-06T09:32:33.405Z]  cannot be applied to (Null, Seq[org.apache.hadoop.fs.FileStatus])

[2022-12-06T09:32:33.405Z] [ERROR]     val pds2 = Seq(PartitionDirectory(null, Seq(fs2)))

[2022-12-06T09:32:33.405Z] [ERROR]                    ^

[2022-12-06T09:32:33.405Z] [ERROR] 7 errors found
```


**Steps/Code to reproduce bug**
build plugin databricks runtimes

--- a/sql-plugin/src/test/scala/com/nvidia/spark/rapids/AlluxioUtilsSuite.scala
+++ b/sql-plugin/src/test/scala/com/nvidia/spark/rapids/AlluxioUtilsSuite.scala
@@ -115,7 +115,7 @@ class AlluxioUtilsSuite extends FunSuite {
     val rapidsConf = new RapidsConf(sqlConf)
     val fs = new FileStatus(0, false, 1, 1024L, 0L,
       new Path("s3a://bucket_1/a.parquet"))
-    val pds = Seq(PartitionDirectory(null, Seq(fs)))
+    val pds = Seq(PartitionDirectory(null, Array(fs)))
     val configuration = new Configuration()
     val runtimeConfig = mock[RuntimeConfig]
     when(runtimeConfig.getOption("spark.hadoop.fs.s3a.access.key")).thenReturn(Some("access key"))
@@ -134,7 +134,7 @@ class AlluxioUtilsSuite extends FunSuite {
     val rapidsConf = new RapidsConf(sqlConf)
     val fs = new FileStatus(0, false, 1, 1024L, 0L,
       new Path("s3a://not_found/a.parquet"))
-    val pds = Seq(PartitionDirectory(null, Seq(fs)))
+    val pds = Seq(PartitionDirectory(null, Array(fs)))
     val configuration = new Configuration()
     val runtimeConfig = mock[RuntimeConfig]
     when(runtimeConfig.getOption("spark.hadoop.fs.s3a.access.key")).thenReturn(Some("access key"))
@@ -154,7 +154,7 @@ class AlluxioUtilsSuite extends FunSuite {
     val rapidsConf = new RapidsConf(sqlConf)
     val fs = new FileStatus(0, false, 1, 1024L, 0L,
       new Path("s3a://bucket_1/dir1/a.parquet")) // matches 2 rules
-    val pds = Seq(PartitionDirectory(null, Seq(fs)))
+    val pds = Seq(PartitionDirectory(null, Array(fs)))
     val configuration = new Configuration()
     val runtimeConfig = mock[RuntimeConfig]
     when(runtimeConfig.getOption("spark.hadoop.fs.s3a.access.key")).thenReturn(Some("access key"))
@@ -179,7 +179,7 @@ class AlluxioUtilsSuite extends FunSuite {
     val rapidsConf = new RapidsConf(sqlConf)
     val fs = new FileStatus(0, false, 1, 1024L, 0L,
       new Path("s3a://bucket_1/dir1/a.parquet")) // matches 2 rules
-    val pds = Seq(PartitionDirectory(null, Seq(fs)))
+    val pds = Seq(PartitionDirectory(null, Array(fs)))
     val configuration = new Configuration()
     val runtimeConfig = mock[RuntimeConfig]
     when(runtimeConfig.getOption("spark.hadoop.fs.s3a.access.key")).thenReturn(Some("access key"))
@@ -202,7 +202,7 @@ class AlluxioUtilsSuite extends FunSuite {
     val rapidsConf = new RapidsConf(sqlConf)
     val fs = new FileStatus(0, false, 1, 1024L, 0L,
       new Path("s3a://bucket_1/a.parquet"))
-    val pds = Seq(PartitionDirectory(null, Seq(fs)))
+    val pds = Seq(PartitionDirectory(null, Array(fs)))
     val configuration = new Configuration()
     val runtimeConfig = mock[RuntimeConfig]
     when(runtimeConfig.getOption("spark.hadoop.fs.s3a.access.key")).thenReturn(Some("access key"))
@@ -228,13 +228,13 @@ class AlluxioUtilsSuite extends FunSuite {
     assert(alluxioFSMock.getMountPoints().isEmpty)
     val fs = new FileStatus(0, false, 1, 1024L, 0L,
       new Path("s3a://bucket_1/a.parquet"))
-    val pds = Seq(PartitionDirectory(null, Seq(fs)))
+    val pds = Seq(PartitionDirectory(null, Array(fs)))
     assert(AlluxioUtils.autoMountIfNeeded(rapidsConf, pds, configuration, runtimeConfig).isDefined)
     assert(alluxioFSMock.getMountPoints().contains("/bucket_1"))
 
     val fs2 = new FileStatus(0, false, 1, 1024L, 0L,
       new Path("myScheme://bucket_2/a.parquet"))
-    val pds2 = Seq(PartitionDirectory(null, Seq(fs2)))
+    val pds2 = Seq(PartitionDirectory(null, Array(fs2)))
     assert(AlluxioUtils.autoMountIfNeeded(rapidsConf, pds2, configuration, runtimeConfig).isEmpty)
     assert(alluxioFSMock.getMountPoints().size == 1)
   }

